src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/BrevModul/Brevskjema.tsx

```diff
@@ -191,7 +191,7 @@ const Brevskjema = ({ onSubmitSuccess }: IProps) => {
             />
             <Fieldset
                 error={
-                    hentFrontendFeilmelding(skjema.submitRessurs) ||
+                    (skjema.visFeilmeldinger && hentFrontendFeilmelding(skjema.submitRessurs)) ||
```

Comment:
Hvorfor skjekker vi her om `skjema.visFeilmeldinger` men ikke på neste linje? 🤔

Reply:
Du mener at det burde vært:

`  error={
                    skjema.visFeilmeldinger && (hentFrontendFeilmelding(skjema.submitRessurs) ||
                    hentFrontendFeilmelding(hentetDokument))
                }`

i stedet (legg merke til parentes forskjellen)?

Kan si meg enig i at det virker tilsynelatende mer logisk - dette var noe jeg fikk hjelp av Kristine med ettersom hele feilhåndteringen var nokså kompleks og ukjent for meg - så jeg ga det ikke så mye ettertanke der og da - da det den doble feilmeldingen som var problemet forsvant og det tilsynelatende virket som forventet. 

`hentFrontendFeilmelding(hentetDokument))` (neste linje som du referer til) var slik det var - så om det er feil å ikke inkludere "skjema.visFeilmeldinger" også i det tilfellet er det ikke en mangel som ble innført med denne commit - men selvfølgelig fint å rette det om det er feil

Sendte spm til Kristine i slack - avventer tilbakemelding der før jeg ev forsker videre på det..

Reply:
Feilmeldingen fra `hentetDokument` er vel uavhengig av om skjemaet skal vise feilmeldinger, så den trenger ikke å ligge bak `skjema.visFeilmeldinger` på samme måte som vi gjør med feilmeldingen fra `skjema.submitRessurs`

Reply:
> Du mener at det burde vært:
> 
> ` error={ skjema.visFeilmeldinger && (hentFrontendFeilmelding(skjema.submitRessurs) || hentFrontendFeilmelding(hentetDokument)) }`
> 
> i stedet (legg merke til parentes forskjellen)?

Nei, jeg mener ikke det burde vært det, vil bare forstå hva som er grunnen til at vi gjør det på den ene men ikke den andre 😊

> 
> Kan si meg enig i at det virker tilsynelatende mer logisk - dette var noe jeg fikk hjelp av Kristine med ettersom hele feilhåndteringen var nokså kompleks og ukjent for meg - så jeg ga det ikke så mye ettertanke der og da - da det den doble feilmeldingen som var problemet forsvant og det tilsynelatende virket som forventet.
> 
> `hentFrontendFeilmelding(hentetDokument))` (neste linje som du referer til) var slik det var - så om det er feil å ikke inkludere "skjema.visFeilmeldinger" også i det tilfellet er det ikke en mangel som ble innført med denne commit - men selvfølgelig fint å rette det om det er feil
> 
> Sendte spm til Kristine i slack - avventer tilbakemelding der før jeg ev forsker videre på det..

Ok, fint at du spurte noen om hvorfor det er sånn 😊

Reply:
Takk for oppklaringen Kristine.